scripts/platforms/greenhouse_apply_v2.py

```
@@ -0,0 +1,550 @@
+
+api_key = os.getenv("GROQ_API_KEY") or os.getenv("OPENAI_API_KEY") or os.getenv("OPENAI_API_KEY_ALT")
+if not api_key:
+    print("Warning: no API key found. AI features will be disabled.")
+    client = None
+else:
+    client = OpenAI(
+        api_key=api_key,
+        base_url="https://api.groq.com/openai/v1"
+    )
+
+with open("applicant.json", "r", encoding="utf-8") as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)